zad1/main.rb

```diff
@@ -1,21 +1,22 @@
-def sort_letters(string)
+#Zadanie 1
+
+#Czesc 1
 
+def sort_letters(string)
+	string.chars.sort.join
 end
 
-def count_vowels(string)
+#Czesc 2
 
+def count_vowels(string)
+	string.count "aeiouy"
 end
 
-def filter_odd(array)
+#Czesc 3
 
+def filter_odd(array)
+	array.select{|x| x.odd?} 
```

Comment:
To nie zmienia działania, ale można tutaj zaaplikować pokazany na zajęciach shorthand bloku. Coś w stylu `array.select(&:odd?)` 😉 przy okazji [w rubym używamy 2 spacji do wcięć](https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation).

Czyli same stylowe pierdoły - poza tym 👍 👍